package.json

```diff
@@ -33,10 +33,10 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
+    "@angular/cli": "^1.0.0",
     "@angular/compiler-cli": "2.4.0",
```

Comment:
@harshit98 reverting back to deprecated packages not a solution for fixing failing builds (exceptions are when new version bring about breaking changes and we're forced to stick to old packages). Otherwise, it's important we stay updated.

@13nehu thanks for the update on the deprecation. The probably reason you're getting errors is the `karma.conf.js` is still referring to the old `angular-cli` package. You'll have to edit it to refer to the new `@angular/cli` package. Use this file (https://gist.github.com/niranjan94/4c1d683d9300bf9f28c3a4d159c462a0) as reference and edit the `karma.conf.js`file. That should fix the error and at the same time we'll be using the updated version of `@angular/cli`